src/assets/scripts/game/GameClockView.js

```diff
@@ -0,0 +1,118 @@
+import { digits_integer } from '../utilities/radioUtilities';
+import { SELECTORS } from '../constants/selectors';
+import { TIME } from '../constants/globalConstants';
+
+/**
+ * Manages a clock that stays in sync with the current game time
+ * @class GameClockView
+ */
+export default class GameClockView {
+    /**
+     * @for GameClockView
+     * @constructor
+     */
+    constructor($element) {
+        this.$element = $element;
+        this.startTime = 0;
+        this.time = 0;
+
+        return this._init($element);
+    }
+
+    /**
+    * Get current time in the user's time zone
+    * @for GameClockView
+    * @property realWorldCurrentLocalTime
+    * @return {number} ms since 01/01/1970, 00:00:00 (user's time zone)
+    */
+    get realWorldCurrentLocalTime() {
+        return new Date().getTime();
+    }
+
+    /**
+    * Get current zulu time in milliseconds
+    * @for GameClockView
+    * @property realWorldCurrentZuluTime
+    * @return {number} ms since 01/01/1970, 00:00:00 UTC
+    */
+    get realWorldCurrentZuluTime() {
+        const date = new Date();
+        const utc = date.getTime() + (date.getTimezoneOffset() * TIME.ONE_MINUTE_IN_MILLISECONDS);
+
+        return utc;
+    }
+
+    /**
+    * Generates a string of the current game time in a human-readable format
+    * @for GameClockView
+    * @property timeString
+    * @return {string} current game time formatted like '03:44:17'
+    */
+    get timeString() {
+        const gameTime = window.gameController.game.time;
+        const clockDate = new Date(this.startTime + (gameTime * TIME.ONE_SECOND_IN_MILLISECONDS));
+        const hours = digits_integer(clockDate.getHours(), 2);
+        const minutes = digits_integer(clockDate.getMinutes(), 2);
+        const seconds = digits_integer(clockDate.getSeconds(), 2);
+        const clockTime = `${hours}:${minutes}:${seconds}`;
```

Comment:
I feel like this should just be a method or two instead of a getter.  

> The get syntax binds an object property to a function that will be called when that property is looked up.

from [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get)